skipper.env

```diff
@@ -193,3 +193,5 @@ IMAGES_FLAVOR
 SERVICE_ACCOUNT_CLIENT_ID
 SERVICE_ACCOUNT_CLIENT_SECRET
 OCM_SELF_TOKEN
+OCM_CLI_REFRESH_TOKEN
+SECOND_OCM_CLI_REFRESH_TOKEN
```

Comment:
Why ?

Reply:
In kni-assisted repo we have this 
`SECOND_OFFLINE_TOKEN = utils.get_env('SECOND_OFFLINE_TOKEN')
SECOND_PULL_SECRET = utils.get_env('SECOND_PULL_SECRET')`

Since offline token is deprecated, we will need a second ocm token.

Reply:
OK so please add a comment why we need it

Reply:
above the var